### ISSUE TITLE
Filter out playlists.

### ISSUE THREAD

**Issue by JonnyCodewalker** [FreeTubeApp/FreeTube#82]
I use FreeTube to keep track of my subscriptions. I watch some lets players and, considering that i am never interested in all the games they play, it would be awesome if there was the ability to filter out certain playlists from the subscription feed.

Like rightclicking on a video and selecting "exclude playlist". That way one would have the capability to keep the inbox a bit more organized.

In order to keep the list of blocked playlists short it could be set so that if there is no update to a playlist in X amount of days it gets unblocked.

**Comment by PrestonN** [comment]
Closing in favor of #95
